### ISSUE TITLE
"lr_schedule" option ignored using torch framework and PPO algorithm

### ISSUE THREAD

**Issue by duburcqa** [ray-project/ray#8368]
*Ray version and other system information (Python version, TensorFlow version, OS):*
- Ray: [0.9.0.dev0 (2c599dbf05e41e338920ee2fbe692658bcbec4dd)](https://s3-us-west-2.amazonaws.com/ray-wheels/releases/0.8.5/02c1ab0ec6d615ad54ebf33bd93c51c04000534e/ray-0.8.5-cp36-cp36m-manylinux1_x86_64.whl)
- CUDA: 10.1
- Pytorch: 1.4.0 with GPU support
- Ubuntu 18.04
- Python 3.6

### What is the problem?

Setting the hyperparameter "lr_schedule" as no effect when using PyTorch as backend framework and PPO learning algorithm.

### Reproduction (REQUIRED)

```
import ray 
from ray.rllib.agents.ppo import PPOTrainer, DEFAULT_CONFIG 
 
config = DEFAULT_CONFIG.copy() 
for key, val in { 
    "env": "CartPole-v0", 
    "num_workers": 0, 
    "use_pytorch": False, 
    "lr": 1.0e-5, 
    "lr_schedule": [ 
        [0, 1.0e-6], 
        [1, 1.0e-7], 
    ] 
}.items(): config[key] = val 
 
ray.init() 

for use_pytorch in [False, True]: 
    config["use_pytorch"] = use_pytorch 
    agent = PPOTrainer(config, "CartPole-v0") 
    for _ in range(2): 
        result = agent.train() 
        print(f"use_pytorch: {use_pytorch} - Current learning rate: "\
              f"{result['info']['learner']['default_policy']['cur_lr']}")
```

- [x] I have verified my script runs in a clean environment and reproduces the issue.
- [x] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/latest/installation.html).

**Comment by sven1977** [comment]
Thanks for filing this issue @duburcqa . Will take a look ...

**Comment by ericl** [comment]
cc @krfricke

**Comment by janblumenkamp** [comment]
I can confirm this - would appreciate an update on it

**Comment by janblumenkamp** [comment]
Looks like the lr and entropy coeff schedule mixin was not added in the Torch PPO policy. This simple PR fixes it:
```
use_pytorch: False - Current learning rate: 9.999999747378752e-06
use_pytorch: False - Current learning rate: 1.0000000116860974e-07
use_pytorch: True - Current learning rate: 1e-05
use_pytorch: True - Current learning rate: 1e-07
```

**Comment by duburcqa** [comment]
Nice, thank you ! The learning rate using tensorflow comes from a conversion from float32 to float64 that must be done somewhere. If you want to check:
```
import numpy as np
print(np.float64(np.float32(1.0e-5)))
print(np.float64(np.float32(1.0e-7)))
```

**Comment by janblumenkamp** [comment]
Hmmm I made some more experiments and I am not convinced that the lr is actually properly updated... Is it possible that the learning rate in `cur_lr` is different from the actual learning rate?
